payments-workflows/src/main/kotlin/com/gevamu/corda/services/XmlService.kt

```diff
@@ -37,26 +50,54 @@ open class XmlService protected constructor(
     protected val serviceHub: AppServiceHub,
     xmlClasses: List<Class<*>>
 ) : SingletonSerializeAsToken() {
+
     protected val jaxbContext: JAXBContext = JAXBContext.newInstance(
-        *(listOf<Class<*>>(CustomerCreditTransferInitiationV09::class.java) + xmlClasses).toTypedArray()
+        *(listOf<Class<*>>(PaymentXmlData::class.java) + xmlClasses).toTypedArray()
     )
-    protected val xmlInputFactory = XMLInputFactory.newFactory()
+    protected val xmlInputFactory: XMLInputFactory = XMLInputFactory.newFactory()
+
+    private val creditTransferInitValidator: XmlValidator
+
+    private val schemaFactory by lazy { SchemaFactory.newInstance(XMLConstants.W3C_XML_SCHEMA_NS_URI) }
```

Comment:
In most cases lazy shouldn't be used in backend applications: for backend applications consistent response time is more desirable than faster start up.